src/App.js

```diff
@@ -1,5 +1,38 @@
+import { Nav, RestrictedRoute } from "components";
+import { Auth, Home, Page404 } from "pages";
+import { useEffect } from "react";
+import { useDispatch } from "react-redux";
+import { Routes, Route } from "react-router-dom";
+import { ToastContainer } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+import { verifyUser } from "reducers/userSlice";
+
 function App() {
-  return <div className="App">Welcome to StarTrivia</div>;
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    const token = localStorage.getItem("startrivia-user-token");
+    if (token) {
+      dispatch(verifyUser(token));
+    }
+  }, []);
+
+  return (
+    <>
+      <Nav />
+
+      <Routes>
+        <Route path="/" element={<Home />} />
+
+        <Route element={<RestrictedRoute />}>
+          <Route path="/auth" element={<Auth />} />
+        </Route>
+
+        <Route path="*" element={<Page404 />} />
+      </Routes>
+      <ToastContainer />
```

Comment:
Appreciation : lottie animations looking great :)  do share the resource how use to them

Reply:
Thanks. Here is the npm package: https://www.npmjs.com/package/react-lottie